libs/langchain-community/src/vectorstores/astradb.ts

```diff
@@ -40,18 +49,35 @@ export class AstraDBVectorStore extends VectorStore {
 
   private readonly contentKey: string; // if undefined the entirety of the content aside from the id and embedding will be stored as content
 
+  private readonly batchSize: number;
```

Comment:
Should set a `batchSize` default since it's optional. Maybe `250`?
```suggestion
  private readonly batchSize = 250;
```

Reply:
default is set in the constructor. 20 is the max supported by `insertMax`
```javascript
this.batchSize = batchSize && batchSize <= 20 ? batchSize : 20;
```